crates/preimage/src/key.rs

```diff
@@ -56,8 +57,9 @@ impl TryFrom<u8> for PreimageKeyType {
 /// |---------|-------------|
 /// | [0, 1)  | Type byte   |
 /// | [1, 32) | Data        |
-#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
```

Comment:
Can we keep `serde` here as well?

Reply:
Maybe we could add a new `rkyv` feature flag? We use `serde` throughout the workspace so it would be nice to keep consistency